modules/client/examples/01-client-token/02-client.ts

```diff
@@ -1,12 +1,12 @@
 /* MARKDOWN
-### Create an ERC20 client
+### Create an Token client
 */
-import { ClientErc20, Context, ContextPlugin } from "@aragon/sdk-client";
```

Comment:
It is consistent to what existed before. Question: would you call it `ClientTokenVoting` instead? I'm fine as it is now

Reply:
we chose now `TokenVotingClient`